8/README.MD

```diff
@@ -241,15 +241,13 @@ To initialize an array you use the square brackets:
 
 `vals = []` - `vals` is an empty list
 
-`vals = [1,2,3]` - `vals` contains the ints `1`, `2` and `3`.
-
-`vals = [1, "aloha", 'c', True, 4.2]`  - 'vals' contains the int `1`, the string `aloha`, the char `c`, the boolean `True` and the float `4.2`.
```

Comment:
While possible, no need to encourage this as a feature. Rarely appropriate to mix types.